src/UserRepositoryInterface.php

```diff
@@ -26,5 +26,13 @@ public function authenticate(string $credential, string $password = null) : ?Use
      * @param string $identity
      * @return string[]
      */
-    public function getRolesFromUser(string $identity) : array;
+    public function getUserRoles(string $identity) : array;
```

Comment:
As asked in #29, why is this in the interface? This could be used internally in a private function in `PdoDatabase`. All the interface should care about is authenticate a user and return a UserInterface object.

Reply:
@xtreamwayz actually you right! I'm refactoring the PR removing these functions. Thanks!